site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -416,32 +416,11 @@ When enabled the user's password hash will be modified if it does not match the
 
 
 ==== Webhooks
-image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - JWT,width=1200,role=shadowed top-cropped]
+image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=top-cropped]
 
-===== Table columns
+===== Form Fields
```

Comment:
While technically part of the form, this is a table on the webhook tab. So the `Event`, `Enabled` and `Transaction setting` are table columns. 

Not sure if we have any other presence for documenting table columns to ensure we keep them consistent?

Reply:
There's only one other place, in application roles. Updated.